server/src/concepts/user/repositories/commands.js

```diff
@@ -0,0 +1,63 @@
+import User from "../model/User";
+
+const createUser = (data) => {
+  const newUser = new User({
+    ...data,
+  });
+
+  try {
+    newUser.save();
+    return newUser;
+  } catch (err) {
+    console.log(err);
+    return "error";
+  }
+};
+
+const updateUser = () => {
+  User.updateOne({}, function (err, result) {
+    if (err) {
+      console.log(err);
+    } else {
+      console.log("Updated: ", result);
+    }
+  });
+};
+
+const updateAllUsers = () => {
+  User.updateMany({}, function (err, result) {
+    if (err) {
+      console.log(err);
+    } else {
+      console.log("Updated: ", result);
+    }
+  });
+};
+
+const deleteUser = () => {
+  User.deleteOne({})
```

Comment:
https://mongoosejs.com/docs/api.html#model_Model.deleteOne
in parameter should object with some key: value pair that indicates what we want to delete
And please change syntax to await instead of callbacks